frontend/components/ResourceInfoCard.tsx

```diff
@@ -0,0 +1,56 @@
+import { Button, Grid, Paper, Sx, Text, rem } from '@mantine/core';
+import Link from 'next/link';
+import React from 'react';
+import { ResourceInfo } from '@/util/types/fhir';
+import { ExternalLink } from 'tabler-icons-react';
+
+export interface ResourceInfoCardProps {
+  resourceInfo: ResourceInfo;
+}
+
+export default function ResourceInfoCard({ resourceInfo }: ResourceInfoCardProps) {
+  return (
+    <Paper
+      shadow="sm"
+      p="md"
+      sx={theme => {
+        const style: Sx = {
+          ':hover': {
+            cursor: 'pointer',
+            backgroundColor: theme.colors.gray[0]
```

Comment:
Right now, we don't do anything when a resource is clicked. I'm not sure if it's more appropriate to remove this hover highlighting, or just make the whole resource card a link to the JSON content.